packages/console/src/components/RoleUsersTransfer/SourceUsersBox.tsx

```diff
@@ -43,11 +44,15 @@ const SourceUsersBox = ({ roleId, selectedUsers, onAddUser, onRemoveUser }: Prop
     };
   }, []);
 
-  const { data } = useSWR<[User[], number], RequestError>(
-    `/api/users?excludeRoleId=${roleId}&page=${pageIndex}&page_size=${pageSize}&hideAdminUser=true${conditionalString(
-      keyword && `&search=${encodeURIComponent(`%${keyword}%`)}`
-    )}`
-  );
+  const url = buildUrl('/api/users', {
+    excludeRoleId: roleId,
+    hideAdminUser: 'true',
+    page: `${pageIndex}`,
+    page_size: `${pageSize}`,
```

Comment:
no need to parse to string if you are using URLSearchParams?

Reply:
The URLSearchParams constructor receives a `Record<string, string>` type param, so we need to parse it to string here.
<img width="832" alt="image" src="https://user-images.githubusercontent.com/10806653/211520510-7c65dfc6-5e3c-47f7-bca8-abf62ffb42d9.png">